net/scripts/gce-provider.sh

```diff
@@ -256,3 +273,31 @@ cloud_FetchFile() {
     gcloud compute scp --zone "$zone" "$instanceName:$remoteFile" "$localFile"
   )
 }
+
+#
+# cloud_CreateAndAttachPersistentDisk [instanceName] [diskSize] [diskType]
+#
+# Create a persistent disk and attach it to a pre-existing VM instance.
+# Set disk to auto-delete upon instance deletion
+#
+cloud_CreateAndAttachPersistentDisk() {
```

Comment:
In the other providers, can you please add a stub `cloud_CreateAndAttachPersistentDisk()` that immediately errors and cleanly exits.

Reply:
done